README.md

```diff
@@ -45,6 +45,34 @@ Past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](h
 If you’d like to join, just go through the [issue list](https://github.com/owncloud/mail/issues) and fix some. :) We’re also in [#owncloud-mail on freenode IRC](https://webchat.freenode.net/?channels=owncloud-mail).
 
 
+## Troubleshooting
+
+### Gmail
+
+In case you are facing issues in accessing your Gmail account you can use https://accounts.google.com/DisplayUnlockCaptcha to unlock your account.
+
+### Outlook.com
+
+In case you are facing issues in accessing your Outlook.com account, you can try to enable the 'Two-Factor Verification' (https://account.live.com/proofs/Manage) and setup an app password (https://account.live.com/proofs/AppPassword), which you then use for the ownCloud mail app.
```

Comment:
Same here: »If you can not access your Outlook.com account …«

Also, »ownCloud Mail app«, Mail is capitalized. ;)